App/Controllers/TiersController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Controllers;
+
+
+class TiersController extends BaseController
```

Comment:
We should consider calling this `Ranks`, ranks have tiers and divisions correct?